Duration: 0.365575 | Epoch 1 | Loss: 0.223705
Duration: 35.673900 | Epoch 101 | Loss: 0.000116
Duration: 37.127455 | Epoch 201 | Loss: 0.000990
Duration: 32.686498 | Epoch 301 | Loss: 0.000313
Duration: 35.666682 | Epoch 401 | Loss: 0.000156
Duration: 36.113180 | Epoch 501 | Loss: 0.000391
Duration: 39.161307 | Epoch 601 | Loss: 0.000229
Duration: 40.685903 | Epoch 701 | Loss: 0.000645
Duration: 33.198223 | Epoch 801 | Loss: 0.000441
Traceback (most recent call last):
  File "/home/chetan/Desktop/CS237B/Homeworks/CS237B-Principles-of-Robot-Autonomy-II/Imitation Learning/train_il.py", line 288, in <module>
    nn_train(data, args,wandb_config_dict)
  File "/home/chetan/Desktop/CS237B/Homeworks/CS237B-Principles-of-Robot-Autonomy-II/Imitation Learning/train_il.py", line 168, in nn_train
    for x_batch, y_batch in train_loader:
                            ^^^^^^^^^^^^
  File "/home/chetan/anaconda3/lib/python3.12/site-packages/torch/utils/data/dataloader.py", line 708, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "/home/chetan/anaconda3/lib/python3.12/site-packages/torch/utils/data/dataloader.py", line 764, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/chetan/anaconda3/lib/python3.12/site-packages/torch/utils/data/_utils/fetch.py", line 52, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
            ~~~~~~~~~~~~^^^^^
  File "/home/chetan/anaconda3/lib/python3.12/site-packages/torch/utils/data/dataset.py", line 211, in __getitem__
    return tuple(tensor[index] for tensor in self.tensors)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/chetan/anaconda3/lib/python3.12/site-packages/torch/utils/data/dataset.py", line 211, in <genexpr>
    return tuple(tensor[index] for tensor in self.tensors)
                 ~~~~~~^^^^^^^
KeyboardInterrupt
Exception ignored in atexit callback: <function _start_and_connect_service.<locals>.teardown_atexit at 0x7f6af1460cc0>
Traceback (most recent call last):
  File "/home/chetan/anaconda3/lib/python3.12/site-packages/wandb/sdk/lib/service_connection.py", line 94, in teardown_atexit
    conn.teardown(hooks.exit_code)
  File "/home/chetan/anaconda3/lib/python3.12/site-packages/wandb/sdk/lib/service_connection.py", line 226, in teardown
    self._router.join()
  File "/home/chetan/anaconda3/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 75, in join
    self._thread.join()
  File "/home/chetan/anaconda3/lib/python3.12/threading.py", line 1149, in join
    self._wait_for_tstate_lock()
  File "/home/chetan/anaconda3/lib/python3.12/threading.py", line 1169, in _wait_for_tstate_lock
    if lock.acquire(block, timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt:
